src/components/Navigation/components/Item/Item.tsx

```diff
@@ -134,6 +159,7 @@ export default class Item extends React.Component<Props, State> {
           >
             {iconMarkup}
             <span className={styles.Text}>{label}</span>
+            {indicatorMarkup}
```

Comment:
Is there anything we can do about the indicator wrapping?

Reply:
Meaning can we prevent it from wrapping while allowing the text to still wrap if needed?

Reply:
Yeah like avoiding the indicator from wrapping below the text, before the text itself wraps. We could try to mitigate this by turning on hyphenation for the text, and having a non-breaking space separate the word and the indicator, but that would need to be tested. I’m also happy to defer this, but it’s worth testing with the menu items we know are going to get this soon, combined with our current set of locales. In other words, it’s OK to defer unless it’s going to immediately look bad to some merchants :)